README.rst

```diff
@@ -1,8 +1,28 @@
-In Video Quiz XBlock |BS| |CA|
-==============================
+In Video Quiz XBlock
+====================
 
 This XBlock allows for edX components to be displayed to users inside of videos at specific time points.
 
+Overview
+--------
+
+This XBlock expects to have a Video component and a Problem component
+added to the same Unit as itself.
+
+The XBlock is then configured (see below) to map timestamps (in seconds)
+to Component IDs (the Problems), along with the Video Component ID.
+
+Whenever video playback reaches a configured timestamp, the video is
+paused and the corresponding Problem Component is overlayed on top of
+the Video.
+
+The user can answer problem, though it is not required.
```

Comment:
nit: answer the problem

question: can you make it required?

Reply:
You can _not_ force it to be required; this was an intentional product choice.